### PR TITLE
Validate length of company name in manifest

### LINKED ISSUE #571: [BUG] Connector fails to load in Tableau 2020.1
**Describe the bug**
When trying to load a connector which is successfully packaged using the `connector_packager`, into Tableau Server 2020.1, it fails to load with the error logs below. (Thanks @pvanderknyff for pointing us to the right logs)
The same connector works without issues in 2019.4
It appears the the company name field was made mandatory starting v2020.1

We believe it would be better to fail the connector packing with a validation error rather than discover the issue from logs while loading the connector in Tableau Server.
Would it be okay if we open a pull request with the proposed changes?

**Logs**
```
{"ts":"2020-07-21T23:48:41.808","pid":11192,"tid":"70c8","sev":"warn","req":"-","sess":"-","site":"-","user":"-","k":"connector-plugin-warning","v":"ReadVendorInfoFromDOM: company name is empty"}
{"ts":"2020-07-21T23:48:41.808","pid":11192,"tid":"70c8","sev":"error","req":"-","sess":"-","site":"-","user":"-","k":"connector-plugin-error","e":{"log-code":"0e9a05c7","log-source":"needs-classification"},"v":"Failed to load manifest.xml from D:\\Tableau_Connectors\\alloy.taco"}
```

**Server (please complete the following information):**
 - OS: Windows Server 2016
 - Tableau Version: Tableau Server 2020.1

**About you:**
 Name: Vignesh
 Company: Goldman Sachs

### PR DESCRIPTION
**Pull request by gs-manvig**
Fixes #571 by validating the length of company name field in manifest.xml

And update CHANGELOG.md

Submitting on behalf of Goldman Sachs

### CODE REVIEW

**Comment by gs-manvig** [comment]
cc @pvanderknyff , please let us know if this looks good

File: validation/connector_plugin_manifest_latest.xsd [hunk 1]
```diff
@@ -172,4 +172,9 @@
   <xs:complexType name="Platform-ST">
     <xs:attribute name="name" type="OSShortName-ST" use="required"/>
   </xs:complexType>
+  <xs:simpleType name="CompanyName-ST">
```
Comment (sxlin) [inline validation/connector_plugin_manifest_latest.xsd:175]: This is better named NonEmptyString-ST, since it might be used for other attributes later.

Comment (gs-manvig) [reply validation/connector_plugin_manifest_latest.xsd]: Renamed. Thanks for the review!

**Review by pvanderknyff — APPROVED** [review]
Looks good to me! Thanks for looking at this issue.

**Review by sxlin — APPROVED** [review]

**Comment by gs-manvig** [comment]
Hi @pvanderknyff , what is the process to get this PR merged and subsequently released.

**Comment by pvanderknyff** [comment]
Merged it, turns out someone at Tableau has to hit the merge button, sorry for that!

**Comment by gs-manvig** [comment]
No worries, thanks!

**Pull request merged**